Add Docker compose section to devcontainer docs (#239)
swift-server/vscode-swift: docs/remote-dev.md

FROM swift:5.5
```

### Using a custom Docker Compose File

> For more complex development environments you might want to use Docker Compose. The `devcontainer.json` file has three settings you need to include if you are wanting to use Docker Compose: 

Reviewer: Avoid passive language
```suggestion
For more complex development environments you may need to use Docker Compose. The `devcontainer.json` file has three settings you need to include if you want to use Docker Compose: 
```

- `dockerComposeFile` your docker compose file
- `service` the service you want to run
- `workspaceFolder` the root folder for your project.

Your `devcontainer.json` should look something like this

```json
{
    "name": "MyService: 5.6-focal",
    "dockerComposeFile": "docker-compose.yml",
    "service": "app",
    "workspaceFolder": "/workspace",
    "extensions": [
      "sswg.swift-lang",
    ],
    "settings": {
      "lldb.library": "/usr/lib/liblldb.so"

>     },

Reviewer: We should probably add a note here about exposing ports as that will be a popular use case
